lib/tasks/test.rake

```diff
@@ -0,0 +1,16 @@
+namespace :test do
+  # Note: you probably want to `> log/test.log` to empty the test log file
+  #  and then re-run tests to get a fresh view
+  desc 'Scan test.log for rendered views'
+  task scan_log_for_render: :environment do
+    # match 'Rendered two_factor_authentication/otp_verification/show.html.erb'
+    # Rendered + space + word + [/ + non-whitespace](any number of times)
+    regex_finder = /Rendered\s\w*(\/\S*)*/
```

Comment:
If we wanted to avoid escaping the forward slash inside the regex, we could switch the outside to pipes or something

```ruby
%r|Rendered\s\w*(/\S*)*|
```
and to make sure it's the same:
```ruby
/Rendered\s\w*(\/\S*)*/ == %r|Rendered\s\w*(/\S*)*|
=> true
```

Reply:
Huh, didn't know about that, thanks!